contracts/Overtime/SportsAMMV2.sol

```diff
@@ -271,21 +289,39 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
         }
 
         if (_collateral != address(0)) {
-            _handleDifferentCollateral(_buyInAmount, _collateral, _isEth, msg.sender);
+            (useLPpool, collateralPriceInUSD) = _handleDifferentCollateral(_buyInAmount, _collateral, msg.sender, _isEth);
+        }
+        if (useLPpool != address(0)) {
```

Comment:
while it doesnt matter for the compiler, for readability this should really be one line of code with ternary operators for differentials parameters

Reply:
gonna delete the check, since `useLPpool` and `collateralPriceInUSD` are address(0) and 0 when a pool is not used

Reply:
done